pages/common/vimdiff.md

```diff
@@ -0,0 +1,24 @@
+# vimdiff
+
+> Open up to four files in Vim and show their differences.
+> See Vim documentation for how to edit and navigate within a window.
+
+- Open files and show the differences:
+
+`vimdiff {{file1}} {{file2}}`
+
+- Use horizontal window split instead of the default vertical split:
+
+`vimdiff -o {{file1}} {{file2}}`
+
+- Close current window without saving changes:
```

Comment:
I'm sorry I'm not clear on what you mean here. Are you saying to mention the ability to use those generic vim commands in that description or to alter the description to allow the reintroduction of those two commands as per previous commits?